src/controllers/user.controller.js

```diff
@@ -7,11 +7,18 @@ const ObjectId = require('mongoose').Types.ObjectId
 const { omit, compact } = require('lodash')
 const bcrypt = require('bcrypt-nodejs')
 
+
+
 exports.create = async (req, res, next) => {
-  try {
+ 
+    try{
+      
+    req.body.name=req.body.name.split(' ').map((word) => {return word[0].toUpperCase()+word.substr(1).toLowerCase() }).join(' ')
```

Comment:
spacing between = is missing. 

Run `gulp lint` or `gulp watch` to see syntax errors

Reply:
If you use VSCode please  install ESLint  with `npm i -g eslint`
Then install ESLint extension for VSCode and enable it.

Then it will show your code style errors

Reply:
i was using vscode disabling eslint :p. now il fix it

Reply:
Sure. Use it to write a clean well standard codebase